pytealext/evaluator/tools.py

```diff
@@ -20,10 +20,10 @@ def compile_and_run(
     mode: Mode = Mode.Application,
     *,
     version: int = MAX_PROGRAM_VERSION,
-    context: EvalContext = None,
-    debug: IO = None,
-    tmpl_subs: dict[str, str] = None
-) -> tuple[list[bytes or int], list[bytes or int]]:
+    context: Optional[EvalContext] = None,
```

Comment:
as of python 3.10 (actually pep 604) instead of `Optional` we should write ` EvalContext | None`

Reply:
It's optional, but sure